cmd/root.go

```diff
@@ -271,7 +274,15 @@ func JsonnetVM(cmd *cobra.Command) (*jsonnet.VM, error) {
 						return nil, fmt.Errorf("Missing environment variable: %s", kv[0])
 					}
 				case 2:
-					spec.setter(kv[0], kv[1])
+					if spec.flagName == flagExtVarEnv {
+						if v, present := os.LookupEnv(kv[0]); present {
```

Comment:
```suggestion
						if v, ok := os.LookupEnv(kv[0]); ok {
```

Not a huge deal, the `ok` for the `bool` in the map is the accepted Go convention for this.